### DISCUSSION TITLE
Forgot to mount usr/app/data folder: How to update and keep projects on dockerized?

### DISCUSSION THREAD

**Discussion by Dont-ask-for-it** [nocodb/nocodb#4888] [Q&A]
I forgot to mount usr/app/data folder on my docker install.

Trying to update without loosing projects. Tried to make a new instance but projects disappers.

Can I just export projects and start on a new version? How to?

I am on 0.98.3 version, with docker on a Synology. 

P.S I tried to export data folder by going in terminal. But I always get this error 👍 
![image](https://user-images.githubusercontent.com/5061749/213572369-2a028c74-5b35-4689-a8ae-5122672e2897.png)

**Comment by wingkwong** [comment]
If you haven't specified `NC_DB`, a default sqlite db called `noco.db` will be created under `/usr/app/data`. You gotta retrieve it first. So far there is no way to export the meta database.

**Comment by Dont-ask-for-it** [comment]
Cant I export metadata and csv ?
